msteams-platform/whats-new.md

```diff
@@ -9,9 +9,11 @@ keywords: teams what's new latest
 >[!TIP]
 > Check out sample solutions to common team scenarios in the [**Teams app template catalog**](samples/app-templates.md)! The catalog is alphabetized and the newest additions are tagged with a star ☆.
 
-## Change Log
+The changes to the Microsoft Teams platform and this document set are listed in the following sections:
+* [General Availability (GA) features](#ga-features): Provides a list of released features in Microsoft Teams. Some entries may be used to call attention to a new feature that is simply of interest to Teams developers.
```

Comment:
```suggestion
* [General availability (GA) features](#ga-features): Microsoft Teams platform features that are available to all app developers.
```
Slight edit since this list is only relevant to Teams app development and not end user or admin features.